support.org

```diff
@@ -2073,3 +2074,24 @@ Open hydra for current major mode if one exists, otherwise the default hydra.
 #+begin_src emacs-lisp
   (call-interactively 'episteme:dashboard)
 #+end_src
+* load user config
+This should remain at the end of this file.
+#+begin_src emacs-lisp
+  (unless (file-exists-p episteme/config)
+    (with-temp-file episteme/config
+      (insert ";; this is your custom configuration for episteme,
+  ;; you can manage it with orgmode by un-commenting
+  ;; the following:
+  ;;
+  ;; (org-babel-load-file
+  ;;  (expand-file-name
+  ;;  \"README.org\"
```

Comment:
Why does this still read README.org

Reply:
It is commented out, as an example for how to load from an orgfile